.travis.yml

```diff
@@ -32,8 +32,8 @@ before_script:
    - wget -c https://inkscape.org/de/gallery/item/14204/Inkscape-883c7bc-x86_64.AppImage -O ~/bin/inkscape.alpha
    - chmod +x ~/bin/inkscape.alpha
    - export PATH=~/bin:$PATH
-   - python test_installation_script.py 2> /dev/null
-   - python setup.py
+   - python3 test_installation_script.py 2> /dev/null
+   - python3 setup.py
```

Comment:
I've moved this into a separate branch, where we can consider more carefully the long term compatibility and maintainability concerns. On my Debian machine, this is still a Python 2 link, but that's mostly a non-issue because I'm working on getting textext into Debian as a package, and I am not using any of these scripts at all.